### ISSUE TITLE
Bigint does not display correctly in Command Log, throws error on print to console of 0n

### ISSUE THREAD

**Issue by FFdhorkin** [cypress-io/cypress#16570]
### Current behavior
There are currently two issues with bigint in cypress.
1. when it prints log output, it says `{}` instead of serializing the bigint. If you click on the assertion, it displays properly in the console:
![image](https://user-images.githubusercontent.com/59899746/118701846-08454900-b7c9-11eb-9e0a-da4b258c72bf.png)
2. If either the expected or actual value is `0n`, you get an  exception when you click on it:
![image](https://user-images.githubusercontent.com/59899746/118702157-6a9e4980-b7c9-11eb-9a50-e1b11e732ea7.png)

### Desired behavior
- Bigints assertions should display their values in the command log (possibly with truncation)
- if you click on a bigint assertion that has `0n` on expected or actual, no exception should occur

### Test code to reproduce
```js
describe('Bigint example', () => {
    it('Displays wrong', () => {
        expect(2n).to.equal(2n);
    });

    it('Exception on inspect', () => {
        expect(1n).to.equal(0n);
    })

    it('Exception on inspect 2', () => {
        expect(0n).to.equal(1n);
    })
});
```
### Versions
This is present in 5.6.0 through 7.3.0, though you get different behavior for the exception in 5.6.0 and 7.3.0. In older versions, it doesn't show the exception and stack  trace, but expected/actual are missing:
![image](https://user-images.githubusercontent.com/59899746/118702995-72122280-b7ca-11eb-8782-8eae09e6d8c9.png)
![image](https://user-images.githubusercontent.com/59899746/118703038-80603e80-b7ca-11eb-89e6-2662bd605b0c.png)

**Comment by jennifer-shehane** [comment]
Our code tries to run lodash `_.trim()` on the value, so `_.trim(0n)` throws that `Cannot mix BigInt` error. Code is here: https://github.com/cypress-io/cypress/blob/develop/packages/runner/src/lib/logger.js#L32:L32

I'm not sure exactly why it displays as `{}` instead of the bigint.

We would accept a pull request to fix the display and error.

**Comment by sainthkh** [comment]
`expect {} to be {}` is happening because of `chai` bug. It's fixed recently in [4.3.4](https://github.com/chaijs/chai/releases/tag/v4.3.4).
